release.json

```
@@ -22,6 +22,30 @@
         "WINDOWS_DDNPM_SHASUM": "5d31cbf7aea921edd5ba34baf074e496749265a80468b65a034d3796558a909e",
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
+    "6.29.0-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.29.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.29.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.29.0-rc.1",
+        "JMXFETCH_VERSION": "0.44.0",
+        "JMXFETCH_HASH": "758717a2fd9c50247336318644e9c985d9a8a9845039d59a8793923b3f76f494",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.10",
+        "MACOS_BUILD_VERSION": "6.29.0-rc.1",
+        "WINDOWS_DDNPM_DRIVER": "release-signed",
+        "WINDOWS_DDNPM_VERSION": "0.98.2.git.74.86e7c5b",
+        "WINDOWS_DDNPM_SHASUM": "61e13fccdca7bf6a356e7ef552a7845316b9905b5eee264811982e3034a3bb84"
```

Could you update these lines with what's used in `nightly`?
```suggestion
        "WINDOWS_DDNPM_VERSION": "0.98.2.git.86.53d1ee4",
        "WINDOWS_DDNPM_SHASUM": "5d31cbf7aea921edd5ba34baf074e496749265a80468b65a034d3796558a909e"
```